### DISCUSSION TITLE
Include a estimated read time in story cards

### DISCUSSION THREAD

**Discussion by triptych** [dailydotdev/daily#681] [Feature Requests]
Many articles lately are simply super short snippets with catchy titles and very little content. It would be a very valuable feature if you were to include an estimated read time in the cards you display for an article - as this can hint if the article is tiny or more substantial. I think you could even use this metric as a filter when folks want to see only longer articles in their "feed".

**Comment by idoshamun** [comment]
Hi @triptych,

That's a very interesting idea. Thanks for sharing :)
